restaurantgame.js

```diff
@@ -0,0 +1,292 @@
+    game[0]= "key"
+    player.Inventory.pop()
+    console.log("You have dropped the key")
+    return play()
+  }
+  else if (action === "drop the" && placePersonItem === "key") {
+    console.log("You don't have the key")
+    return play()
+  }
+  else if (action === "read the" && placePersonItem === 'sign') {
+    console.log(itemTable[placePersonItem].read())
+    return play()
+  }
+  else if (action === "read the" && placePersonItem === 'menu') {
+    console.log(itemTable[placePersonItem].read())
+    return play()
+  }
+  else if (action === "take the" && placePersonItem === 'key') {
+    console.log(itemTable[placePersonItem].take())
+    return play()
+  }
+  else if (action === "take the" && placePersonItem === 'tip') {
+    console.log(itemTable[placePersonItem].take())
+    return process.exit()
+  }
+  else if (action === "eat the" && placePersonItem === "fries") {
+    console.log(itemTable[placePersonItem].eat())
+    return play()
+  }
+  if (action === "drink the" && placePersonItem === 'fiddlehead') {
+    console.log(itemTable[placePersonItem].eat())
+    return play()
+  }
+  else if (action === "go on" && placePersonItem === "hike") {
+    console.log("You decide to skip your first day of work- YOU ARE FIRED!")
+    process.exit()
+  }
+  else if (action === "check my" && placePersonItem === "inventory") {
+    console.log("You are carrying: " + player.Inventory)
+    return play()
+  }
+  else if (action === "check my" && placePersonItem === "status") {
+    console.log(player.status)
+    return play()
+  }
+  else if (action === "check the" && placePersonItem === "game") {
+    console.log(game)
+    return play()
+  }
+  else if (action === "exit the" && placePersonItem === "game") {
+    process.exit()
+  }
+  else {
+    console.log("Sorry I do not recognize that!! Try again")
+    return play()
+  }
+}
+
+console.log("Welcome to your first day of work, there are a few things to remember to keep you caught up. \n\n To talk to a person, please type, talk to 'individuals-name'.\n To take an item, please type, take the 'item-name'.\n To eat an item, please type eat the 'item-name'.\n To drink an item, please type drink the 'item-name' \n To change rooms, please type go to 'room name'\nTo check your inventory, please type check my inventory\nTo check your status, please type check my status\n\n\n 133 Bank St. \n\n Walking towards the lake, you pass friendly faces standing outside of Alf's diner on the left.\n Damn! You've forgotten to eat lunch again, you're famished and need some food soon.\n 2:50 PM.\n Your shift starts at 3:00.\nYour stomach has butterflies as you realize you're not as early as you planned on.\n'Last first day of a restaurant job,' you say to yourself, as you cross St.Paul Street headed towards, 'One Rock.'\nAs you turn to go to work, you see your friend Sam running up St. Paul Street flagging you down.` ")
+play()
```

Comment:
You have a well thought out game, however, other than thievery, there is no exit from the game, like my shift never ends, which is triggering some old restaurant nightmares I used to have when I worked in the industry!